website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-azure-devops.md

```diff
@@ -7,7 +7,7 @@ sidebar_label: "Connecting Azure DevOps"
 <Snippet src="available-beta-banner" />
 
 :::info
-You can still use dbt Cloud with Azure DevOps before this feature is generally available by [following these instructions](/dbt-cloud/cloud-configuring-dbt-cloud/cloud-import-a-project-by-git-url#azure-devops).
+You can still use dbt Cloud with Azure DevOps before this feature is generally available by [following these instructions](/dbt-cloud/cloud-configuring-dbt-cloud/cloud-import-a-project-by-git-urll#azure-devops).
```

Comment:
Is the URL supposed to be this? I did _not_ test out the following suggestion:
```suggestion
You can still use dbt Cloud with Azure DevOps before this feature is generally available by [following these instructions](docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-import-a-project-by-git-url#azure-devops).
```

Reply:
Just tested and Doug's change works for me!